src/installer/CMakeLists.txt

```diff
@@ -90,4 +90,18 @@ install(FILES "${PROJECT_ROOT}/THIRD-PARTY" DESTINATION doc COMPONENT "Docs")
 # Install resource files
 install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/Resources/opensearch_sql_odbc.tdc" DESTINATION resources COMPONENT "Resources")
 
+if(WIN32)
+    set(AWS_SDK_BIN_DIR "${PROJECT_ROOT}/src/vcpkg_installed/x${BITNESS}-windows/bin")
+    install(DIRECTORY ${AWS_SDK_BIN_DIR} DESTINATION . COMPONENT "Driver" FILES_MATCHING PATTERN "*.dll")
+
+    if(CMAKE_BUILD_TYPE STREQUAL "Debug")
```

Comment:
is this input parameter by the user?  Should we check case-insensitive?

Reply:
Maybe. I just recovered deleted piece of file. The only thing I added is ` FILES_MATCHING PATTERN "*.dll"` to exclude big `pdb` files (debug symbols).